### DISCUSSION TITLE
Fuzzy finder in buffer

### DISCUSSION THREAD

**Discussion by nickel8448** [zed-industries/zed#13851] [Feature Requests]
Hello, is it possible to have a fuzzy finder for text in the buffer? Thanks

**Comment by SomeoneToIgnore** [comment]
It's not.

**Reply by SomeoneToIgnore** [reply]
One thing that might help a bit is the "outline: toggle"  command, that will open a modal with every tree-sitter outline capture, this could be filtered with fuzzy matches:
![Screenshot 2024-07-05 at 14 06 19](https://github.com/zed-industries/zed/assets/2690773/bd52f264-9aa5-4a97-9753-5e5f8e06f99d)

**Reply by nickel8448** [reply]
Unfortunately that will not suffice. Thank you for taking out to the time to answer my question.

**Comment by failable** [comment]
I miss the commands like `consult-lines` in Emacs for fuzzy grepping lines.

<img width="963" alt="image" src="https://github.com/zed-industries/zed/assets/5475262/2c5d2840-3425-41d4-9a90-9736982d6736">
